connector/spanmetricsconnector/connector.go

```diff
@@ -79,6 +77,8 @@ type resourceMetrics struct {
 	sums       metrics.SumMetrics
 	events     metrics.SumMetrics
 	attributes pcommon.Map
+	// startTimestamp captures when the first data points for this resource are recorded.
+	startTimestamp pcommon.Timestamp
```

Comment:
Is this related to the fix?

Reply:
Yes, it is related @atoulme. As the sums can now go back to `0` for the same attribute combination under cumulative temporality, the start timestamp needs to be reset according to the spec:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#temporality